test/unit/user_test.rb

```diff
@@ -231,23 +231,25 @@ def setup_user(operation)
   test "non-admin user can delegate roles he has assigned already" do
     setup_user "create"
     create_role          = Role.find_by_name 'create_users'
-    record               = User.new :login    => "dummy", :mail => "j@j.com", :auth_source_id => AuthSourceInternal.first.id,
-                                    :role_ids => [create_role.id.to_s]
+    record               = User.new :login    => "dummy", :mail => "j@j.com", :auth_source_id => AuthSourceInternal.first.id
     record.password_hash = "asd"
     assert record.valid?
```

Comment:
the valid assertion is pointless, though, as assert save validates and returns false in case object's not valid